### PR TITLE
Use classlist instead of appending and removing class names

### PR DESCRIPTION
**Pull request by mitogh**
This was causing and issue when the same item was selected multiple times the same element it was having the `is-selected` class multiple times.

screen shot 2017-02-11 at 11 40 58 pm

### CODE REVIEW

File: blocks.js [hunk 1]
```diff
@@ -122,10 +122,13 @@ function showControls( node ) {
 
 	// show/hide block-specific block controls
 	var kinds = getTypeKinds( blockType );
-	var kindClasses = kinds.map( function( kind ) {
+	kinds.map( function( kind ) {
 		return 'is-' + kind;
-	} ).join( ' ' );
-	blockControls.className = 'block-controls ' + kindClasses;
+	} )
+	.forEach( function( className ) {
```
Comment (mcsf) [inline blocks.js]: For consistency, could you join the two above lines into a single `} ).forEach( …`?

**Review by mcsf — COMMENTED** [review]
Thanks for jumping in! I left a very minor observation. Feel free to make that tweak, then ping me and I'll merge this.

File: blocks.js (hunk 1 above)
Comment (mitogh) [inline blocks.js]: Sure.

**Comment by mitogh** [comment]
Changes has been added to the PR @mcsf

**Comment by mcsf** [comment]
👌

**Pull request merged**